dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathBuilderTest.java

```diff
@@ -70,7 +70,7 @@ public void testArtifactsToPaths_removingDuplicates() throws RepositoryException
     Path opencensusApiPath = opencensusApiPathFound.get();
     Truth.assertWithMessage("Opencensus API should have multiple dependency paths")
         .that(multimap.get(opencensusApiPath).size())
-        .isGreaterThan(1);
+        .isEqualTo(1);
```

Comment:
This change is because `DependencyGraphBuilder.levelOrder` does not visit same node twice.